src/custom/utils/price.ts

```diff
@@ -408,6 +112,8 @@ export async function getGpUsdcPrice({ strategy, quoteParams }: Pick<LegacyQuote
       quoteToken: quoteParams.sellToken,
     }
 
+    const { getBestPrice } = await import('utils/priceLegacy')
```

Comment:
Warning! Since the `LEGACY` strategy is an edge case, we don't need to load `0x`, `paraswap` and `1inch` modules always.